lib/kramdown-rfc2629.rb

```diff
@@ -494,13 +500,18 @@ def convert_img(el, indent, opts) # misuse the tag!
         if a = el.attr
           alt = a.delete('alt').strip
           alt = '' if alt == '!' # work around re-wrap uglyness
-          if anchor = a.delete('src')
-            a['target'] = anchor
+          if src = a.delete('src')
+            a['target'] = src
           end
         end
         if alt == ":include:"   # Really bad misuse of tag...
+          anchor = el.attr.delete('anchor')
+          if !anchor
+            warn "*** missing anchor for '#{src}'"
+            anchor = src
+          end
           to_insert = ""
-          anchor.scan(/(W3C|3GPP|[A-Z-]+)[.]?([A-Za-z_0-9.-]+)/) do |t, n|
+          src.scan(/(W3C|3GPP|[A-Z-]+)[.]?([A-Za-z_0-9.\/\+-]+)/) do |t, n|
```

Comment:
Yeah, I think that requires sanitizing the cache filename.  The anchor should be sanitized already.